README.md

```diff
@@ -2,35 +2,18 @@
 
 [![Build/test](https://github.com/derekstride/tree-sitter-sql/actions/workflows/ci.yml/badge.svg)](https://github.com/derekstride/tree-sitter-sql/actions/workflows/ci.yml)
 
-SQL grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter).
+A general/permissive SQL grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter).
 
 ## Features
 
-- [x] SELECT statements (some features may be incomplete)
-- [x] Function calls
-- [x] Index hints
-- [x] Comments
-- [x] Marginalia
-- [x] UPDATE statements
-- [x] DELETE statements
-- [x] CREATE statements
-- [x] INSERT/REPLACE statements
-- [x] DROP statements
-- [x] ALTER TABLE statements
-- [ ] CREATE INDEX statements
-- [ ] DROP INDEX statements
-- [ ] ALTER INDEX statements
-- [ ] EXPLAIN statements
-- [ ] Function definitions
-
 For a complete list of features see the the [tests](test/corpus)
```

Comment:
I do like having the list but it'll take some time and care to produce a really useful one and keep it up to date, so this is fine for now imo